poprox-db/migrations/versions/2024_12_11_1611-c2007fa35fc2_add_caption_column_to_images_table.py

```diff
@@ -0,0 +1,26 @@
+"""add caption column to images table
+
```

Comment:
It doesn't seem like you have the pre-commit hooks installed, which would raise these issues when you try to commit. You can install them by running `pre-commit install` in each repo directory.